JsonLD.Infrastructure.Text.Tests/JsonLD.Infrastructure.Text.Tests.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
would prefer netstandard, but for now not wanting to change the core test project which is netcoreapp

Reply:
Yeah, let's have `netstandard` as a goal, but we'll get there when we'll get there.